### ISSUE TITLE
How does trino support unsigned int64, if you query mysql's unsigned int64 will it overflow

### ISSUE THREAD

**Issue by china-shang** [trinodb/trino#14676]
io.trino.spi.type.LongDecimalType  Will it be used to support this case ?

**Comment by tangjiangling** [comment]
> io.trino.spi.type.LongDecimalType Will it be used to support this case ?

Yes, when reading a MySQL `unsigned bigint` column, Trino converts it to `Decimal(20, 0)` (which internally in Trino is represented by `LongDecimalType`).
